presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestJdbcConnection.java

```diff
@@ -105,21 +105,86 @@ public void teardownServer()
     public void testCommit()
             throws SQLException
     {
-        try (Connection connection = createConnection()) {
-            connection.setAutoCommit(false);
-            try (Statement statement = connection.createStatement()) {
-                statement.execute("CREATE TABLE test_commit (x bigint)");
+        try {
```

Comment:
It's hard to see in github, but the only thing that's changed about this test is that it's wrapped in an extra try so I could put a finally at the end to drop the table.

Reply:
A neat trick is to take the url in the browser and add the query parameter `?w=1`, which will cause whitespaces to be ignored.  e.g. https://github.com/prestodb/presto/pull/23453/files?w=1